content/en-us/characters/emotes.md

```diff
@@ -45,17 +45,17 @@ local humanoidDescription = humanoid.HumanoidDescription
 
 -- Set custom emotes within a table
 local emoteTable = {
-  ["Hello"] = {3576686446},
-  ["Stadium"] = {3360686498},
-  ["Tilt"] = {3360692915},
-  ["Shrug"] = {3576968026},
-  ["Salute"] = {3360689775},
-  ["Point"] = {3576823880}
+	["Hello"] = { 3576686446 },
+	["Stadium"] = { 3360686498 },
+	["Tilt"] = { 3360692915 },
+	["Shrug"] = { 3576968026 },
+	["Salute"] = { 3360689775 },
+	["Point"] = { 3576823880 },
 }
 humanoidDescription:SetEmotes(emoteTable)
 
 -- Equip emotes in a specific order
-local equippedEmotes = {"Hello", "Stadium", "Tilt", "Shrug", "Salute", "Point"}
+local equippedEmotes = { "Hello", "Stadium", "Tilt", "Shrug", "Salute", "Point" }
```

Comment:
```suggestion
local equippedEmotes = {"Hello", "Stadium", "Tilt", "Shrug", "Salute", "Point"}
```

Reply:
I don't prefer the spaces between first and last elements of a table, since I would wonder why not put space between (for example) the parameters of a function too (i.e. `local function foo( "arg1", arg2 )`.